### PR TITLE
add: `nuclear-deb`

### PR DESCRIPTION
**Pull request by AloneER0**
image
Now everything should be correct 👀

### CODE REVIEW

File: packages/nuclear-deb/nuclear-deb.pacscript [hunk 1]
```diff
@@ -0,0 +1,9 @@
+name="nuclear-deb"
+gives="nuclear"
+breaks="${gives} ${gives}-app"
+version="e8fcb6"
+description="Streaming music player that finds free music for you "
```
Comment (Henryws) [inline packages/nuclear-deb/nuclear-deb.pacscript:5]:
```suggestion
description="Streaming music player that finds free music for you"
```

**Review by Henryws — CHANGES_REQUESTED** [review]

File: packages/nuclear-deb/nuclear-deb.pacscript [hunk 2]
```diff
@@ -0,0 +1,9 @@
+name="nuclear-deb"
+gives="nuclear"
+breaks="${gives} ${gives}-app"
+version="e8fcb6"
```
Comment (Henryws) [inline packages/nuclear-deb/nuclear-deb.pacscript:4]: Are the versions based on hashes?

Comment (AloneER0) [reply packages/nuclear-deb/nuclear-deb.pacscript:4]: Yes

Comment (AloneER0) [reply packages/nuclear-deb/nuclear-deb.pacscript:4]: There aren't any releases only tags

Comment (Henryws) [reply packages/nuclear-deb/nuclear-deb.pacscript:4]: Ugh, I'm not sure that we can have non number starting versions.

Comment (AloneER0) [reply packages/nuclear-deb/nuclear-deb.pacscript:4]: Hmm as in? since pacstall installed it fine

Comment (Henryws) [reply packages/nuclear-deb/nuclear-deb.pacscript:4]: But if you upgrade it, it won't work.

Comment (AloneER0) [reply packages/nuclear-deb/nuclear-deb.pacscript:4]: Oh.

Comment (Henryws) [reply packages/nuclear-deb/nuclear-deb.pacscript:4]: Seems that the latest version is 0.6.17

Comment (AloneER0) [reply packages/nuclear-deb/nuclear-deb.pacscript:4]: Yes but downloading the deb package doesn't have a tag with that version

Comment (Henryws) [reply packages/nuclear-deb/nuclear-deb.pacscript:4]: Then set version to `0.6.17`, then create `_version` and use that one in place of `${version}`

Comment (AloneER0) [reply packages/nuclear-deb/nuclear-deb.pacscript:4]: So another variable?

Comment (AloneER0) [reply packages/nuclear-deb/nuclear-deb.pacscript:4]:
```name="nuclear-deb"
gives="nuclear"
breaks="${gives} ${gives}-app"
version="0.6.17"
_version="e8fcb6"
description="Streaming music player that finds free music for you "
hash="10b3d9fafbe07ad9feb89747c6cf06e3b2288f021d197f2880042434774e8008"
maintainer="AloneER0 <aloneer0@outlook.com>"
url="https://github.com/nukeop/nuclear/releases/download/${_version}/nuclear-${_version}.deb"
repology=("project: ${gives}")``` like this?

File: packages/nuclear-deb/nuclear-deb.pacscript [hunk 3]
```diff
@@ -0,0 +1,10 @@
+name="nuclear-deb"
+gives="nuclear"
+breaks="${gives} ${gives}-app"
+version="0.6.17"
+_version="e8fcb6"
+description="Streaming music player that finds free music for you "
```
Comment (Henryws) [inline packages/nuclear-deb/nuclear-deb.pacscript:6]:
```suggestion
description="Streaming music player that finds free music for you"
```

**Review by Henryws — CHANGES_REQUESTED** [review]

**Review by Henryws — APPROVED** [review]

**Pull request merged**